packages/python/readme_metrics/PayloadBuilder.py

```diff
@@ -95,12 +99,14 @@ def _build_request_payload(self, request: Request) -> dict:
         post_data = {}
         headers = None
 
+        # Convert EnivronHeaders to a dictionary
+        headers_dict = dict(request.headers.items())
         if self.blacklist:
```

Comment:
Should this be `self.denylist`?

Reply:
it's possible I missed this one, let me check

Reply:
yeah missed the headers for the python lib, good catch